### ISSUE TITLE
an Illustrator error occurred: 1128353364 ('TNAC')

### ISSUE THREAD

**Issue by l-i-n-k** [iconic/illustrator-svg-exporter#13]
I have switched to Illustrator CC 2015 (19.1.0) and get this error when trying to export.
Anybody experiencing the same issue?

**Comment by chrisoshea** [comment]
Yeah I get this issue too.  A shame as was looking for Generator for Illustrator :)
